packages/openactive-integration-tests/test/helpers/flow-stages/flow-stage-utils.js

```diff
@@ -55,14 +55,15 @@ const FlowStageUtils = {
    *   validation checks.
    * @param {ValidationMode} validationSpec.validationMode
    */
-  simpleValidationTests(logger, { name, validationMode }) {
-    return (/** @type {FlowStageTypeWithHttpResponseOutput} */ flowStage) => {
+  simpleValidationTests(logger, { name, validationMode}) {
+    return (/** @type {FlowStageTypeWithHttpResponseOutput} */ flowStage, /** @type {boolean} */doValidateInErrorMode) => {
       sharedValidationTests.shouldBeValidResponse(
         () => flowStage.getOutput().httpResponse,
         name,
         logger,
         {
           validationMode,
+          doValidateInErrorMode,
```

Comment:
Fly-by comment from a cursory glance... 

Might be worth clarifying this as `doValidateInOrderItemErrorMode` just to distinguish this from other types of errors